src/main/java/uk/gov/hmcts/juror/api/moj/service/PoolCreateServiceImpl.java

```diff
@@ -501,7 +506,7 @@ private JurorPool createJurorPool(String login, String owner, Voters voter,
         juror.setResponded(false);
         juror.setContactPreference(null);
 
-        juror = jurorRepository.saveAndFlush(juror);
+        //juror = jurorRepository.save(juror);
```

Comment:
Why do we no longer save the juror here?

Reply:
This is because the save is done later after we have created the new entity. Removed comment.